src/VirtualClient/VirtualClient.Actions/SPECview/SpecViewExecutor.cs

```diff
@@ -126,15 +183,16 @@ protected override async Task ExecuteAsync(EventContext telemetryContext, Cancel
                 string[] viewsetArray = this.Viewset.Split(new[] { "," }, StringSplitOptions.RemoveEmptyEntries);
                 string specviewOriginalLogPath;
                 string specviewRenamedLogPath;
-                List<Task> tasks = new List<Task>();
+                var tasks = new List<Task>();
+
                 foreach (string viewset in viewsetArray)
                 {
                     // log file is inside a folder that starts with the viewset name e.g. 3dsmax-07
-                    string? viewsetLogDir = this.fileSystem.Directory.GetDirectories(this.historyResultsPath, $"{viewset}*", SearchOption.TopDirectoryOnly).FirstOrDefault();
+                    string? viewsetLogDir = this.PlatformSpecifics.Combine(this.historyResultsPath, this.viewsetLogFileNameMapping[viewset]);
```

Comment:
I think the logdir would not be empty with the this.PlatformSpecifics.Combine.

Reply:
You are right. Removed the check. If the log dir does not exist, then this.fileSystem.Directory.Move will throw directorynotfound expection anyways.